### ISSUE TITLE
<video> tag in markdown doesn't respect the blog posts container

### ISSUE THREAD

**Issue by lkopacz** [forem/forem#1885]
**Describe the bug**
I crosspost all my blog posts from my personal blog and in the most recent particular post I used a lot of video tags that were not using YouTube or Vimeo.

**To Reproduce**

1. Go to https://dev.to/new and create a new post.
2. In the Markdown, add ` video tag like the one below 
```
<video controls style="max-width: 590px;">
  <source src="https://www.a11ywithlindsey.com/marcy-sutton-clapping-tweet-1fca86b197aff683ffd0a940e70e5737.mp4" type="video/mp4">
</video>
```
3. In the Front Matter, keep `published: false`, but add all necessary fields save the post so you can video the page.
4. Go to the post, and see the video. It extends past the blog post container.

**Expected behavior**
A video should have a max-width of the container. 

**Screenshots**
<img width="1251" alt="screen shot 2019-02-25 at 7 28 57 am" src="https://user-images.githubusercontent.com/6540866/53337945-67f56800-38d0-11e9-88dd-6d6c1c3a36d4.png">

**Desktop (please complete the following information):**

- OS: Mac Mojave
- Browser: Chrome Version 72.0.3626.109
- Version: 10.14.3

**Comment by Link2Twenty** [comment]
I think we'd need to add `controls` to the scrubber's `attributes` list
https://github.com/thepracticaldev/dev.to/blob/master/app/sanitizers/rendered_markdown_scrubber.rb

Then in the CSS, we'd want something like.
```css
video {
  max-width: 100%;
}
```
https://github.com/thepracticaldev/dev.to/blob/master/app/assets/stylesheets/article-show.scss

**Comment by jessleenyc** [comment]
hm I didn't even realize we supported the video tag...but looking at the code we clearly do. This is approved if anyone wants to work on it -- @Link2Twenty seems to have the right idea.

**Comment by mariocsee** [comment]
I can work on this if no one is on it!

**Comment by lkopacz** [comment]
Thanks @mariocsee! Can someone @ me when the PR gets merged? I have a post in drafts that I've been waiting to post 🙃

**Comment by Zhao-Andy** [comment]
We'll want to fix this bug before we enable native `<video>` tag embeds. For now, we suggest using the YouTube or Vimeo Liquid tag embeds for video embeds. More details in [our editor guide.](https://dev.to/p/editor_guide)

Related comments in this PR: https://github.com/thepracticaldev/dev.to/pull/1928

**Comment by hsablonniere** [comment]
Hello team :wink:

I have the same need as @lkopacz
I have automated cross posting articles from my site to dev.to and I really want to host and embed my own content without using YouTube or Vimeo.

I saw in this thread and in https://github.com/forem/forem/pull/1928 that this is not considered but I'm not sure I understand why. Especially since it almost works. It just needs the width styles and the controls attribute.

If there's another solution that involves forem's platform but not YouTube/Vimeo, I would be interested ;-)
